packages/react-components/src/components/MessageThread.tsx

```diff
@@ -51,6 +49,7 @@ import { FileMetadata } from './FileDownloadCards';
 /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
 import { AttachmentDownloadResult } from './FileDownloadCards';
 import { useTheme } from '../theming';
+import { createV9Theme, useFluentV9Wrapper } from './../theming/v9ThemeShim';
```

Comment:
When will this be available upstream?

Reply:
I will update this to a separate object (as you advised) in the next PR